frontend/src/pages/settings/logs-and-activity.vue

```diff
@@ -136,98 +132,128 @@ import IMdiStop from 'virtual:icons/mdi/stop';
 import IMdiHelp from 'virtual:icons/mdi/help';
 import { useDateFns, useRemote } from '@/composables';
 
-interface LoadingStatus {
-  status: 'loading' | 'loaded' | 'error';
-  errorMessage?: string;
+type LoadingStatus<T> =
+  | { status: 'loading' }
+  | { status: 'loaded'; data: T }
+  | { status: 'error'; message?: string };
```

Comment:
I actually like this as a concept in certain cases, it's a more clear representation of what the state can be based on the status.
That being said, I agree that it's overkill and not useful here, since like you said we block the page while loading anyway. I'll rip it out :)

Reply:
The one thing this does hide is the error state on the template, which is nice that it has. I'm a little split, it's easy to rip this all out, and if there's an error loading the logs, then just an empty list will be shown, which isn't the best UX.
On the other hand, we could just not await those initial data calls (or put them in an immediate only watcher), but the loading state is also just a blank list which isn't super clear for a user either and would require some more thought.

I'll get rid of it all for now along with translations for the error state in a separate commit just so it's easy to revert if we do want to keep it.

Reply:
@aweebs I think the most reusable approach would be to always rely in Suspense for rendering components that are fast enough to load (not the case of library pages, for instance) and just show a generic snackbar if the load fails. Having each page handle their state like this is a nightmare for inconsistency.

The probability of errors in this component is really really low. Anyway, we have this somehow implemented with the Axios interceptors, but I have a local branch where I'm working in getting a progress bar at the top (in a similar fashion to YouTube), I can expand it for better global exception handling.